### ISSUE TITLE
Is Speaker Diarization Available or Planned?

### ISSUE THREAD

**Issue by Abandersen04** [chidiwilliams/buzz#961]
I’m currently using Buzz for transcribing interviews with multiple speakers. However, I’ve noticed that the transcription doesn’t differentiate between different voices or speakers in the audio. Is speaker diarization (speaker identification) available or on the roadmap as a feature?

Additionally, I noticed the "prompt" feature, but it doesn't seem to affect speaker recognition. Could you clarify its purpose and if it might relate to this?

Thanks in advance for your help!

**Comment by raivisdejus** [comment]
Yes this is one of idea that has been requested previously. It is in the list of things that would be nice to have in the future. 
Also seems that https://github.com/MahmoudAshraf97/whisper-diarization works quite well, so it could be implemented in the Buzz at some future day.

**Comment by adijahangir123** [comment]
Isn't possible to utilize pyannote.audio as it is claiming to show good results.

**Comment by raivisdejus** [comment]
@Abandersen04 @adijahangir123 @snoopy3841 I have started thinking about this feature. Please see this discussion https://github.com/chidiwilliams/buzz/discussions/1043 and share your intended workflow so we can figure out the best way to implement it.

Implementation of the feature will be tracked in https://github.com/chidiwilliams/buzz/issues/492

**Comment by raivisdejus** [comment]
@Abandersen04 @adijahangir123 @snoopy3841 Speaker identification has been implemented and is available in latest development version https://chidiwilliams.github.io/buzz/docs/faq#9-where-can-i-get-latest-development-version
